returnn/tf/util/data.py

```diff
@@ -2337,6 +2337,32 @@ def copy_extend_batch(self, batch):
         data.placeholder = x
     return data
 
+  def copy_unpack_batch(self):
```

Comment:
This copies only the template. The function name should reflect that.
```suggestion
  def copy_template_unpack_batch(self):
```